connector/src/test/java/com/google/cloud/spark/bigquery/AvroSchemaConverterTest.java

```diff
@@ -32,6 +32,8 @@
 import java.math.BigDecimal;
 import java.nio.ByteBuffer;
 
+import static com.google.cloud.spark.bigquery.SchemaConverters.*;
```

Comment:
Please avoid `import static Class.*`, import the actual fields/methods

Reply:
Done